Makefile

```
@@ -317,6 +317,11 @@ test-yaml:
 	@# Fall back to find if there is no git, e.g. in package builds
 	yamllint --strict $$((git ls-files "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml') | grep -v ^dbicdh)
 
+.PHONY: test-shfmt
+test-shfmt:
+	@which shfmt >/dev/null 2>&1 || (echo "Command 'shfmt' not found, can not execute bash script syntax checks" && false)
+	shfmt -d -d -i 4 -bn -ci -sr $(shfmt -f . | grep -v '^.bpan' )
```

I think this doesn't work as it is a Makefile, where shell syntax doesn't work.
Also we don't have .bpan here in this repo, so maybe don't use it.
Try this:
```suggestion
	shfmt -d -d -i 4 -bn -ci -sr $$(shfmt -f .)
```
With the double dollar it should work.